README.md

```diff
@@ -1,4 +1,48 @@
-HtUserRegistrationDoctrineORM
-=============================
-
-Doctrine2 ORM storage adapter for HtUserRegistration.
+HtUserRegistrationDoctrineORM
+=============================
+
+This module extends [HtUserRegistration](https://github.com/hrevert/HtUserRegistration)
+by replacing the Zend DB code with Doctrine.
+
+Install
+-------
+
+Require with composer
+```
+composer require hrevert/ht-user-registration-doctrine-orm
+```
+
+Include in config/application.config.php
+```php
+    'modules' => array(
+        ...
+        'HtUserRegistration',
+        'HtUserRegistrationDoctrineORM',  // include after HtUserRegistration
+        ...
+    ),
+```
+
+
+Configure
+----------
+
+Your User entity must implement `HtUserRegistrationDoctrineORM\Entity\UserInterface`.

```

Comment:
I don't believe this module should be considered "less equal" than any other module.  If you want to implement a module which extends the User' data then it should require no less than another module which extends the User' data.  Just because the data this implements is small makes it no less important in the Doctrine Metadata.